students/piotrowski_stanislaw/lesson_01_basics/L1_IFN_05.py

```diff
@@ -0,0 +1,14 @@
+#https://snakify.org/lessons/integer_float_numbers/problems/digit_after_decimal_point/
+#piotrsta
+
+import math
+
+print('Podaj dodatnia liczbe rzeczywista: ')
+num = float(input())
+
+frac, whole = math.modf(num)
+frac2=str(frac)
```

Comment:
użyj operacji matematycznych aby to osiągnąć, zamiast konwersji na string.

Reply:
Poprawiłem.